Geometry/HGCalCommonData/interface/HGCalGeomRotation.h

```diff
@@ -0,0 +1,15 @@
+#ifndef Geometry_HGCalCommonData_HGCalGeomRotation_h
+#define Geometry_HGCalCommonData_HGCalGeomRotation_h
+
+class HGCalGeomRotation {
+public:
+  HGCalGeomRotation();
+  ~HGCalGeomRotation() {}
+
+  enum WaferCentring { WaferCentred, CornerCentredY, CornerCentredMercedes };
```

Comment:
prefer to use enum class (then unknown values just aren't allowed)

Reply:
Changed to enum class